lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -83,53 +94,99 @@ class DataReductionFITSWriter(Tool):
     > lstchain_create_dl3_file
         -d /path/to/DL2_data_file.h5
         -o /path/to/DL3/file/
-        --input-irf /path/to/irf.fits.gz
+        --input-irf-path /path/to/irf/
+        --irf-file-pattern irf*.fits.gz
+        --final-irf-file final_interp_irf.fits.gz
         --source-name Crab
         --source-ra 83.633deg
         --source-dec 22.01deg
-        --global-gh-cut 0.9
         --overwrite
 
     Or generate source-dependent DL3 files
     > lstchain_create_dl3_file
         -d /path/to/DL2_data_file.h5
         -o /path/to/DL3/file/
-        --input-irf /path/to/irf.fits.gz
+        --input-irf-path /path/to/irf
+        --irf-file-pattern irf.fits.gz
         --source-name Crab
         --source-dep
         --overwrite
+
+    Or use a list of IRFs for including interpolated IRF:
+    > lstchain_create_dl3_file
+        -d /path/to/DL2_data_file.h5
+        -o /path/to/DL3/file/
+        -i /path/to/irf/
+        -p irf*.fits.gz
+        -f final_interp_irf.fits.gz
+        --interp-method linear
+        --source-name Crab
+        --source-ra 83.633deg
+        --source-dec 22.01deg
+        --overwrite
+
+    Or use a list of IRFs for including only the nearest IRF:
+    > lstchain_create_dl3_file
+        -d /path/to/DL2_data_file.h5
+        -o /path/to/DL3/file/
+        -i /path/to/irf/
+        -p irf*.fits.gz
+        -f final_interp_irf.fits.gz
+        --use-nearest-irf-node True
```

Comment:
This example doesn't work. The True is not expected since use-nearest-irf-node is a flag.

Reply:
True. Will fix it.